packages/jose/jose.0.7.0/opam

```diff
@@ -0,0 +1,56 @@
+opam-version: "2.0"
+synopsis: "JOSE implementation for OCaml and ReasonML"
+description:
+  "JavaScript Object Signing and Encryption built ontop of pure OCaml libs"
+maintainer: ["ulrik.strid@outlook.com"]
+authors: ["Ulrik Strid"]
+license: "MIT"
+homepage: "https://ulrikstrid.github.io/reason-jose"
+doc: "https://ulrikstrid.github.io/reason-jose"
+bug-reports: "https://github.com/ulrikstrid/reason-jose/issues"
+depends: [
+  "ocaml" {>= "4.08.0"}
+  "base64" {>= "3.0.0"}
+  "dune" {>= "2.8" & >= "1.11"}
+  "eqaf" {>= "0.7"}
+  "mirage-crypto" {>= "0.10.0"}
```

Comment:
```suggestion
  "mirage-crypto" {>= "0.10.4"}
```
The current signature failure come from rresult used by ppx_sexp_conv.
We could try to add a confliuct with rresult < 0.7.0 or, if it is not a problem with you, by moving to mirage-crypto 0.10.4, you don't need ppx_sexp_conv any longer and it should solve the issue

Reply:
That sounds reasonable to me, can't fix this from my phone but if you have time please make the change 😁